### PR TITLE
[O11y][ActiveMQ] Update README

### LINKED ISSUE #8861: [O11y][ActiveMQ] Update README
Need to update README with all data stream overviews and Jolokia setup steps to collect metrics.

### Related Issues

- https://github.com/elastic/integrations/issues/6048

### PR DESCRIPTION
**Pull request by harnish-elastic**
## Proposed commit message

- Update README with more details of ActiveMQ and all the data streams that supported.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have added an entry to my package's `changelog.yml` file.

## Related issues

- Closes #8861

### CODE REVIEW

File: packages/activemq/_dev/build/docs/README.md [hunk 1]
```diff
@@ -1,25 +1,100 @@
...
+These logs are System logs of ActiveMQ.
 
 {{event "log"}}
 
 {{fields "log"}}
 
 ### Audit Logs
 
-Audit logs collects the ActiveMQ Audit logs.
+For many secured environments, there's a requirement to log every user management action. For that ActiveMQ implements audit logging, which means that every management action made through JMX or Web Console management interface will be logged and available for later inspection.
```
Comment (alaudazzi) [inline packages/activemq/_dev/build/docs/README.md:97]:
```suggestion
In secured environments, it is required to log every user management action. ActiveMQ implements audit logging, which means that every management action made through JMX or Web Console management interface is logged and available for later inspection.
```

File: packages/activemq/_dev/build/docs/README.md [hunk 2]
```diff
@@ -29,23 +104,23 @@ Audit logs collects the ActiveMQ Audit logs.
...
+ActiveMQ brokers serve as implementations of the Java Messaging Service (JMS), a Java specification facilitating the seamless exchange of data between applications. Metrics provide insights into statistics such as enqueued and dequeued messages, as well as details on consumers, producers, and memory usage (broker, store, temp).
 
 {{event "broker"}}
 
 {{fields "broker"}}
 
 ### Queue Metrics
 
-The server queue stream collects data from the ActiveMQ queue module.
+Queues are FIFO (first-in, first-out) pipelines of messages produced and consumed by brokers and clients. Producers create messages and push them onto these queues. Then those messages are polled and collected by consumer applications, one message at a time. Metrics show statistics of exchanged messages, consumers, producers and memory usage.
```
Comment (alaudazzi) [inline packages/activemq/_dev/build/docs/README.md:115]:
```suggestion
Queues are FIFO (first-in, first-out) pipelines of messages produced and consumed by brokers and clients. Producers create messages and push them onto these queues. Then, those messages are polled and collected by consumer applications, one message at a time. Metrics show statistics of exchanged messages, consumers, producers and memory usage.
```

File: packages/activemq/docs/README.md [hunk 3]
```diff
@@ -103,7 +178,7 @@ An example event for `log` looks as following:
 
 ### Audit Logs
 
-Audit logs collects the ActiveMQ Audit logs.
+For many secured environments, there's a requirement to log every user management action. For that ActiveMQ implements audit logging, which means that every management action made through JMX or Web Console management interface will be logged and available for later inspection.
```
Comment (alaudazzi) [inline packages/activemq/docs/README.md:181]:
```suggestion
In many secured environments, it is required to log every user management action. For that, ActiveMQ implements audit logging, which means that every management action made through JMX or Web Console management interface is logged and available for later inspection.
```

File: packages/activemq/docs/README.md [hunk 4]
```diff
@@ -382,7 +457,7 @@ An example event for `broker` looks as following:
 
 ### Queue Metrics
 
-The server queue stream collects data from the ActiveMQ queue module.
+Queues are FIFO (first-in, first-out) pipelines of messages produced and consumed by brokers and clients. Producers create messages and push them onto these queues. Then those messages are polled and collected by consumer applications, one message at a time. Metrics show statistics of exchanged messages, consumers, producers and memory usage.
```
Comment (alaudazzi) [inline packages/activemq/docs/README.md:460]:
```suggestion
Queues are FIFO (first-in, first-out) pipelines of messages produced and consumed by brokers and clients. Producers create messages and push them onto these queues. Then, those messages are polled and collected by consumer applications, one message at a time. Metrics show statistics of exchanged messages, consumers, producers and memory usage.
```

**Review by alaudazzi — APPROVED** [review]
Left a few editing suggestions, otherwise LGTM.

**Review by aliabbas-elastic — APPROVED** [review]
LGTM

File: packages/activemq/_dev/build/docs/README.md [hunk 5]
```diff
@@ -1,11 +1,86 @@
...
+
+Here is the breakdown of the pattern:
+
+- %-5p: This part represents the log level left-aligned with a width of 5 characters. The - signifies left alignment.
+
+- %m: This part represents the log message.
+
+- %t%n: This part represents the thread name (%t) followed by a newline (%n).
+
+### ActiveMQ
```
Comment (muthu-mps) [inline packages/activemq/_dev/build/docs/README.md:62]: We need to change this title to either Logs or ActiveMQ Logs. Please change this based on the naming conventions followed below for Logs datastream.

File: packages/activemq/changelog.yml [hunk 6]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 0.16.0
+  changes:
+    - description: Update README.
```
Comment (muthu-mps) [inline packages/activemq/changelog.yml:4]:
```suggestion
    - description: Update README to use documentation guidelines.
```

**Review by muthu-mps — APPROVED** [review]
LGTM!

**Pull request merged**